File: CMOWebApi.Models/AdminModel/MasterModel/ProjectSchemeCategoryMasterModel.cs
using System;

namespace CMOWebApi.Models.AdminModel.MasterModel
{
    public class ProjectSchemeCategoryMasterModel
    {
        public int Id { get; set; }
        public Nullable<int> NodalDepartmentCode { get; set; }
        public Nullable<long> ProgramSchemeTypeCode { get; set; }
        public string ProgramSchemeName { get; set; }
        public bool IsActive { get; set; } = true;
        public bool IsDeleted { get; set; } = false;
    }

    public class ProjectSchemeCategoryMasterViewModel
    {
        public int Id { get; set; }
        public Nullable<int> Code { get; set; }
        public Nullable<int> NodalDepartmentCode { get; set; }
        public Nullable<long> ProgramSchemeTypeCode { get; set; }
        public string ProgramSchemeName { get; set; }
        public bool IsActive { get; set; }
        public string DepartmentTitle { get; set; }
        public string ProgramSchemeTypeName { get; set; }
		public Nullable<int> TotalProjectsAdded { get; set; }
	}
}

File: ANGULAR/ANGULAR/src/app/Shared/Model/Master/project-scheme-category-master-model.ts
export class ProjectSchemeCategoryMasterModel {
    Id: number;
    NodalDepartmentCode: string|number;
    ProgramSchemeTypeCode: string|number;
    ProgramSchemeName: string;
    IsActive: boolean;
    IsDelete: boolean;
}

export class ProjectSchemeCategoryMasterViewModel {
  Id: number;
  Code: number;
  NodalDepartmentCode: string|number;
  ProgramSchemeTypeCode: string|number;
  ProgramSchemeName: string;
  DepartmentTitle: string;
  ProgramSchemeTypeName: string;
  TotalProjectsAdded: string|number;
  IsActive: boolean;
}
